### PR TITLE
Pin zip_tricks to 5.3.1 in order to avoid breaking download all...

### LINKED ISSUE #2320: Argo download all files option fails with "Secure Connection Failed"
**Describe the bug**
The download all files option on an Argo show page fails with the following error (in Firefox, but Chrome also fails to download):

```
Secure Connection Failed

An error occurred during a connection to argo.stanford.edu. PR_CONNECT_RESET_ERROR

    The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the website owners to inform them of this problem.
```

**User Impact**
It's not possible to download all files from an object with a single click.

**To Reproduce**
1. Go to an Argo item that has files, e.g. https://argo.stanford.edu/view/druid:bb003dn0409
2. Click on download all files (in the right hand side of the "contents" header)
3. Wait
4. Continue to wait
5. See error

**Expected behavior**
A zip should start downloading after a few seconds.

**Additional context**
Could this be the result of something changing after the UIT maintenance shut down? I just noticed the issue this week, but I don't use it that often so I don't remember when it last worked for me.

### PR DESCRIPTION
**Pull request by aaron-collier**
## Why was this change made?

Fixes #2320 

## How was this change tested?

Manually on prod

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -57,7 +57,7 @@ gem 'rake'
 gem 'retries'
 gem 'ruby-prof'
 gem 'rubyzip'
-gem 'zip_tricks', '~> 5.3'
+gem 'zip_tricks', '5.3.1'
```
Comment (mjgiarlo) [inline Gemfile:60]: Can you add a comment here about the pin?

**Review by mjgiarlo — APPROVED** [review]
Feel free to merge once you stuff a comment in the Gemfile. Thanks! Good work!

File: Gemfile (hunk 1 above)
Comment (aaron-collier) [reply Gemfile]: Done.

**Comment by jcoyne** [comment]
@aaron-collier have you reported this bug to the maintainer of zip_tricks?  Would you please do that and add a comment in the gemfile pointing at the issue?

**Comment by mjgiarlo** [comment]
@jcoyne @aaron-collier do we know enough to write up an actionable, informative issue?

**Comment by jcoyne** [comment]
@mjgiarlo I that that even letting them know that our use case broke between version X and version Y is helpful

**Comment by mjgiarlo** [comment]
> @mjgiarlo I that that even letting them know that our use case broke between version X and version Y is helpful

🆗

**Comment by jcoyne** [comment]
Here are the commits to check. https://github.com/WeTransfer/zip_tricks/compare/v5.3.1...v5.4.0 We could be extra helpful and bisect these.

**Comment by jcoyne** [comment]
I'll bet you a 🍸  that it's https://github.com/WeTransfer/zip_tricks/commit/ccd64dbf25084235892e5f4bba82fb9ca7c13ef6

**Comment by mjgiarlo** [comment]
> I'll bet you a  that it's [WeTransfer/zip_tricks@ccd64db](https://github.com/WeTransfer/zip_tricks/commit/ccd64dbf25084235892e5f4bba82fb9ca7c13ef6)

That's the commit I was most suspicious of, so not taking the bet.

**Pull request merged**